jscomp/others/jsxDOM.res

```diff
@@ -0,0 +1,621 @@
+/* Copyright (C) 2015-2016 Bloomberg Finance L.P.
```

Comment:
Bloomberg Finance does not seem right to me.

@cristianoc What should the copyright header be like for new files?

Maybe
```
Copyright (C) 2022- Authors of ReScript
```

or something like that?

Reply:
That looks good

Reply:
Ok, then we should probably also add this to the CONTRIBUTING doc, too.